spec/menu_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'menu'
+describe Menu do
+  let(:russian) { described_class.new }
+
+  it "displays the menu" do
```

Comment:
You can make your tests easier to read by describing the method, and then using it blocks to document the expected behaviour.